Makefile

```
@@ -1,5 +1,5 @@
-check: githooks
-	cargo check --no-default-features
+check-no-std: githooks
+	cd example && cargo check --no-default-features && cd ..
```

```suggestion
	cd example && cargo check --no-default-features
```

make spawn new shell so no need cd back